### DISCUSSION TITLE
Problème de chargement de tarteaucitron.js

### DISCUSSION THREAD

**Discussion by AlexisAmand** [AmauriC/tarteaucitron.js#672] [Q&A]
Bonjour,

J'utilise Matomo pour les stats de mes sites web, j'ai donc mis en place tarteaucitron.js pour gérer mes cookies. J'ai téléchargé le fichier JS et suivi le guide d'installation pour intégrer tous ça, mais j'ai l'impression que ça ne fonctionne pas du premier coup. 

Je ne sais pas trop comment expliquer... 

Par exemple, si je vais sur ce site : [http://thegasp.genealexis.fr/](http://thegasp.genealexis.fr),) il faut que je raffraichisse la page avec la touche F5 pour enfin voir la fenêtre de gestion des cookies et/ou le petit logo en bas à droite de l'écran.

Peut-être avez-vous une idée ?
Merci d'avance.

Alexis.

**Comment by AmauriC** [comment]
Hello,
Tout semble pourtant ok. Le bandeau ne s'affiche pas, Matomo étant autorisé par défaut, il n'y a pas besoin d'afficher le bandeau.
Le petit cookie en bas à gauche est bien affiché pour permettre à l'utilisateur de désactiver si besoin.
Il n'y a pas de remonté des stats?

**Reply by AlexisAmand** [reply]
Merci pour la réponse, ça rassure d'avoir un témoignage qui me confirme que "tout est normal". J'ai des remontés de stats comme d'habitude.

J'ai fait des tests de mon côté, ça marche parfaitement avec Edge, Chrome et Firefox mais quand je teste avec Brave (mon navigateur par défaut), il faut que j'appuie sur F5 pour ça fonctionne. C'est peut-être dû au fait que c'est un navigateur qui est très axé sur la sécurité.

**Reply by AmauriC** [reply]
Ou que le site est mis en cache. En supprimant tout le cache sur Brave, même problème?
tarteaucitron n'est pas une menace en terme de sécurité, au contraire ^^

**Reply by AlexisAmand** [reply]
En effet, peut-être un problème de cache. Je vais prendre un peu de recul pendant le weekend et me replongé dedans lundi. Merci encore pour la réponse et l'aide.

**Reply by jonathan-ati** [reply]
Bonjour, 
Avez-pu régler ce problème ?
Jonathan
